universal-login-relayer/src/integration/sql/migrations/20200708000001_create_encrypted_wallets.js

```diff
@@ -0,0 +1,18 @@
+/*
+  THIS IS A MIGRATION
+  This file SHOULD NOT be RENAMED or CHANGED in ANY WAY!
+  If you need to change the database schema create a new migration.
+*/
+
+exports.up = async (knex) => {
+  await knex.schema.createTable('encrypted_wallets', (table) => {
+    table.increments();
+    table.string('walletJSON', 900).notNullable();
+    table.string('email').notNullable();
```

Comment:
In my opinion, email should be the primary key or at least unique value.

Reply:
Done